### PR TITLE
Logos at bottom of Our Work are now svg

### PR DESCRIPTION
**Pull request by aileenjeffries**
re: https://trello.com/c/MXPBRAZ6/198-case-study-logos-should-totes-be-svg

Now, thanks to the magic of Googling and searching through Basecamp, they are!
screen shot 2016-06-24 at 9 51 09 am

@tylersticka @nicolemors @mrgerardorodriguez

### CODE REVIEW

File: src/data/case-studies.yml [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-  - /images/sandbox/logo_entertainment.png
-  - /images/sandbox/logo_hautelook.png
-  - /images/sandbox/logo_ceasefire.png
+  - /images/case-studies/logos/logo_walmart.svg
+  - /images/case-studies/logos/logo_deschutes.svg
+  - /images/case-studies/logos/logo_obama-08.svg
+  - /images/case-studies/logos/logo_treasure-coast-hospice.svg
+  - /images/case-studies/logos/logo_entertainment.svg
+  - /images/case-studies/logos/logo_hautelook.svg
+  - /images/case-studies/logos/logo_ceasefire.svg
```
Comment (tylersticka) [inline src/data/case-studies.yml]: Nit: `logo_` prefixes are redundant if these files are in a `logos` directory.

File: src/static/images/case-studies/logos/logo_walmart.svg [hunk 2]
```diff
@@ -0,0 +1,50 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 215 67">
+<g fill="#000000">
```
Comment (tylersticka) [inline src/static/images/case-studies/logos/logo_walmart.svg]: `fill` defaults to black, so this isn't necessary.

**Comment by tylersticka** [comment]
A couple of inline comments. Some other, broader comments for the assets themselves:

- There are a lot of groups (`<g>`) that don't appear to be serving a purpose. We should remove them.
- At least one of the assets has many occurrences of `<path>` with `fill` attribute set to `none`. That probably means the `path` is invisible, perhaps some garbage data exported from Illustrator. This should be confirmed and the offending paths should be removed.

**Comment by aileenjeffries** [comment]
@tylersticka I renamed the files to remove the `logo_` and cleaned up the svg code, getting rid of all groups and removing those empty filled shapes.

**Comment by tylersticka** [comment]
@aileenjeffries Yay! LGTM 👍

**Pull request merged**